sdk/monitor/monitor-query/package.json

```
@@ -109,7 +109,7 @@
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/abort-controller": "^1.0.0",
     "@azure/identity": "^2.0.1",
-    "@azure/monitor-opentelemetry-exporter": "^1.0.0-beta.10",
+    "@azure/monitor-opentelemetry-exporter": "^1.0.0-beta.11",
```

let's ditch the `^` so this is more obvious in the future
```suggestion
    "@azure/monitor-opentelemetry-exporter": "1.0.0-beta.11",
```